src/bin/wallet.rs

```diff
@@ -302,6 +393,19 @@ fn process_command(
                 println!("Not found");
             }
         }
+        // Make a transaction from raw instruction data
+        WalletCommand::TransferRaw(ref userdata) => {
+            let last_id = client.get_last_id();
+
+            // TODO: Update userdata & transaction specifics vis-a-vis Budget DSL determinations
+            let pubkey_vec: Vec<u8> = userdata.iter().cloned().skip(28).collect();
```

Comment:
What's the `skip(28)` for?

Reply:
A really inelegant way to jump to the `to` Pubkey within the userdata. I can improve, but didn't want to spend too much time on it, as that'll be the first to go when we determine how keys are handled.